curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/set-up-passport.md

```diff
@@ -111,7 +111,7 @@ Session and session secret should be correctly set up.
     (data) => {
       assert.match(
         data,
-        /secret:( |)process.env.SESSION_SECRET/gi,
+        /secret:( |)process\.env(\.SESSION_SECRET|\[(?<q>"|')SESSION_SECRET\k<q>\])/gi,
```

Comment:
```suggestion
        /secret:( |)process\.env(\.SESSION_SECRET|\[(?<q>"|')SESSION_SECRET\k<q>\])/g,
```

variables and method names are case sensitive, the `i` flag would pass stuff that doesn't work